### PR TITLE
[VPU]Added support for 2 axis for MVN layer

### PR DESCRIPTION
**Pull request by PoliOwl**
### Details:
 - This pr adds support for Person-reidentification-retail model on VPU device by adding support for `{2}` axis in MVN layer

### Tickets:
 - CVS-55666

### CODE REVIEW

File: inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp [hunk 1]
```diff
@@ -88,11 +90,13 @@ void FrontEnd::parseMVN(const Model& model, const ie::CNNLayerPtr& layer, const
     for (int i = 0; i < indicesSize; i++) {
         axes.insert(getDimFromAxis(ndims, indicesPtr[i]));
     }
+    const auto width = axes.count(Dim::W);
 
-    VPU_THROW_UNLESS(!axes.count(Dim::N) && axes.count(Dim::H) && axes.count(Dim::W),
```
Comment (DariaMityagina) [inline inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp:92]: Why did you remove check for `Dim::H`?

Comment (PoliOwl) [reply inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp:92]: Because now if `axes.count(Dim::H)` is 0, it means there is only `Dim::W`, and this pr adds support for this case, and if `axes.count(Dim::H)` is 1, there is both  `Dim::W` and  `Dim::H`, whitch is already supported, so there is no need to check for it.

Comment (DariaMityagina) [reply inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp:92]: Okay, thanks

**Review by DariaMityagina — APPROVED** [review]

**Review by KorA6 — APPROVED** [review]
